SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -1443,6 +1461,13 @@ void CDriver::InstantiateSpeciesNumerics(unsigned short nVar_Species, int offset
       numerics[iMGlevel][SPECIES_SOL][source_first_term] = new CSourceNothing(nDim, nVar_Species, config);
     }
     numerics[iMGlevel][SPECIES_SOL][source_second_term] = new CSourceNothing(nDim, nVar_Species, config);
+    if(config->GetKind_Upwind_Species() == UPWIND::BOUNDED_SCALAR){
+          numerics[iMGlevel][SPECIES_SOL][source_first_term]->SetBoundedScalar(true);
+          numerics[iMGlevel][SPECIES_SOL][source_second_term]->SetBoundedScalar(true);
+    }else{
+      numerics[iMGlevel][SPECIES_SOL][source_first_term]->SetBoundedScalar(false);
+      numerics[iMGlevel][SPECIES_SOL][source_second_term]->SetBoundedScalar(false);
+    }
```

Comment:
ok I give up, fix the formating of the code for the love of god

Reply:
I moved the setting of the bounded_scalar boolean to the convective numerics class constructor, so it no longer appears in CDriver.